pkg/controller/import-controller.go

```diff
@@ -471,9 +471,9 @@ func (r *ImportReconciler) createImporterPod(pvc *corev1.PersistentVolumeClaim)
 	if err != nil {
 		return err
 	}
-
+	anno := pvc.GetAnnotations()
```

Comment:
I would suggest making a GetPriorityClass(pvc) in pkg/controller/util.go that for now just reads the annotation and returns the value. Then you can call that to pass to createImporterPod. This centralizes the lookup of the PriorityClass and make it easier to modify later. You can also just call the same function from all the controllers. We are not very consistent with this, but I think it would improve maintenance effort in the future to be easier.

Reply:
Added the util methods